src/test/groovy/org/assertj/generator/gradle/parameter/TemplateChanges.groovy

```diff
@@ -281,20 +261,18 @@ class TemplateChanges {
         File templateFolder = testProjectDir.newFolder('templates')
         File content = templateFolder.toPath().resolve("template.txt").toFile()
         content << NEW_TEMPLATE_CONTENT
-        
+
         def contentPath = testProjectDir.root.toPath().relativize(content.toPath()).toString().replace('\\', '/')
-        
+
         buildFile.delete()
         TestUtils.buildFile(buildFile, """
-            assertJ {
```

Comment:
All of these changes are replacing a "global config" with a source-set specific version.